src/mixins/HighlightMixin.js

```diff
@@ -262,7 +286,68 @@ const createSpanStylesheet = (document, identifier, color) => {
 
   document.documentElement.style.setProperty(variables.color, color);
 
-  const rules = {
+  const rules = isFF(FF_DEV_2786) ? {
```

Comment:
The non-dynamic css rules need to be split out into a css file eventually, i do see that this is something that was included before. We'll likely need to create a technical debt task for later where we create a css class where we can define all the static rules and only keep the dynamic css rules here.

regardless i think that change would be out of scope for this sub-task/PR i would say my above feedback is out of scope

Reply:
It would definitely be nice to do it one way in the whole app.